### DISCUSSION TITLE
二开代码如何引入到项目中

### DISCUSSION THREAD

**Discussion by xujiajun5175** [tusen-ai/naive-ui#6179] [Q&A]
1.下载了最新的zip代码包,然后正常pnpm install && pnpm build:package 但是遇到打包错误 [issue:6178](https://github.com/tusen-ai/naive-ui/issues/6178)
2.dist只生成了4个js文件
3.打包成tar.gz 在项目中引入
4.会显示引入缺少package.json。以及找不到对应的路径。
这个具体要怎么做?

**Comment by Sepush** [comment]
为什么不直接在项目中安装 naive-ui 的 npm 包
